examples/api/lib/material/button_style_button/button_style_button.icon_alignment.0.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'package:flutter/material.dart';
 
+// ignore: deprecated_member_use
```

Comment:
Can we just update the comment?

Reply:
@chunhtai and I chatted offline. `ButtonStyleButton.iconAlignment` is now ignored. We should update this sample to explain `ButtonStyle.iconAlignment`. Tracked by: https://github.com/flutter/flutter/issues/179149